package.json

```diff
@@ -333,9 +335,9 @@
     "@metamask/snaps-rpc-methods": "^12.3.0",
     "@metamask/snaps-sdk": "^6.24.0",
     "@metamask/snaps-utils": "^9.3.0",
-    "@metamask/solana-wallet-snap": "^1.30.4",
+    "@metamask/solana-wallet-snap": "^1.31.2",
     "@metamask/solana-wallet-standard": "^0.4.1",
-    "@metamask/transaction-controller": "^56.1.0",
+    "@metamask/transaction-controller": "patch:@metamask/transaction-controller@npm%3A56.1.0#~/.yarn/patches/@metamask-transaction-controller-npm-56.1.0-b106223e34.patch",
```

Comment:
Introduced by https://github.com/MetaMask/metamask-extension/pull/33349

- on `main` we have txn-controller `56.3.0`  https://github.com/MetaMask/metamask-extension/pull/33282
- Hotfix PR patches txn-controller based on `54.3.0` and now in master https://github.com/MetaMask/metamask-extension/pull/33269/files
- 12.20.0 we have `56.1.0`
=> Hence @vinistevam patched a new version based on `56.1.0`